MIDIchlorians/MIDIchlorians/Colour.swift

```diff
@@ -44,6 +44,28 @@ enum Colour: String {
         }
     }
 
+    var image: UIImageView {
+        var imageView = UIImageView()
+        switch self {
+            case .violet:
+                imageView = UIImageView(image: UIImage(named: "cesiousButton"))
```

Comment:
Yeah..these are not the final colours. I was hoping we can get our designer to make something similar for more pleasant colours. The ones I have now were cropped from this [image](https://us.novationmusic.com/sites/default/files/resources/image/lp-pro-overhead_0.jpg). Not very important right now.